docker/tests/Templates/fxdependent-debian.8/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM debian:jessie
+
+ARG PACKAGENAME
+ARG PACKAGELOCATION
+ARG PREVIEWSUFFIX=
+ARG TESTLIST=/PowerShell/test/powershell/Modules/PackageManagement/PackageManagement.Tests.ps1,/PowerShell/test/powershell/engine/Module
+ARG TESTDOWNLOADCOMMAND="git clone --recursive https://github.com/PowerShell/PowerShell.git"
+
+# Install dependencies
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        apt-utils \
+        ca-certificates \
+        curl \
+        apt-transport-https \
+        locales \
+        git \
+        wget
```

Comment:
don't add `wget`, use the docker `ADD` commande

Reply:
updated.